tests/modules/programs/pandoc/defaults.nix

```
@@ -0,0 +1,30 @@
+{ config, ... }:
+
+let cfg = config.programs.pandoc;
+
+in {
```

Since this test depends on the Pandoc package, which is quite large, it is best to run this only when `enableBig` is set.
```suggestion
in lib.mkIf config.test.enableBig {
```